paywall/src/data-iframe/blockchainHandler/index.js

```diff
@@ -162,4 +154,12 @@ export async function listenForAccountNetworkChanges({
       onChange({ keys: {}, transactions: {} })
     }
   })
+
+  await ensureWalletReady(walletService)
```

Comment:
It looks we are calling functionality similar  to that which exists in side of `pollForAccountChange`. Would it be worthwhile for us to call `pollForAccountChange` and eliminate the potential special case of code here?

Reply:
The code below propagates initial state to the cache. I wonder ... It is possible that this was never called because of the bug this PR fixes, and it may be removable. I'll investigate in a quick follow-up PR so we can get this rolling as-is

Reply:
I think, after a quick look, that there is a very good chance this is unneeded, and I'm going to quickly try it out

Reply:
some stuff breaks, going to leave this for now with a TODO to investigate